auth/jwt.go

```diff
@@ -0,0 +1,53 @@
+package auth
+
+import (
+	"gopkg.in/square/go-jose.v2"
+)
+
+// SupportedAlgorithms is list of signing algorithms explicitly supported
+// by github.com/coreos/go-oidc package. If a provider supports other algorithms,
+// such as HS256 or none, those values won't be passed to the IDTokenVerifier.
+var SupportedAlgorithms = map[jose.SignatureAlgorithm]bool{
+	jose.RS256: true,
+	jose.RS384: true,
+	jose.RS512: true,
+	jose.ES256: true,
+	jose.ES384: true,
+	jose.ES512: true,
+	jose.PS256: true,
+	jose.PS384: true,
+	jose.PS512: true,
+}
+
+// JWTConfigCommon groups together configuration options common to both OIDC and local JWT authentication.
+type JWTConfigCommon struct {
+	Issuer         string  `json:"issuer"`                    // OIDC Issuer
+	Register       bool    `json:"register"`                  // If true, server will register new user accounts
+	ClientID       *string `json:"client_id,omitempty"`       // Client ID
```

Comment:
This may end up being renamed to `audience` for local-JWT (still discussing with PM)